src/server/generators/tag.js

```diff
@@ -62,8 +62,13 @@ export default function tagGenerator ({ ssrAppId, attribute, tagIDKeyName } = {}
           attrs += ` ${prefix}${attr}` + (isBooleanAttr ? '' : `="${tag[attr]}"`)
         }
 
+        let json = ''
+        if (tag.json) {
+          json = JSON.stringify(tag.json)
```

Comment:
This content is _still_ directly being rendered to HTML. Should we call it `unsafeJson` or internally escape special values from both keys and values?

Reply:
PS: I couldn't find a serializer only package in the NPM market. We could create and maintain a low overhead one ourselves.

Reply:
Hmm yeah, thanks for the keen eye. I think it should be save if we would use VueMeta's sanitizers and just walk the JSON object to escape all its values. Actually I was half expecting that to work already, looking into that now

Reply:
[escape](https://github.com/nuxt/vue-meta/blob/master/src/shared/escaping.js#L22) seems like good util. However, let's not forget to also sanitize keys :)